.php_cs

```diff
@@ -86,38 +87,37 @@ $config->setRules(
         'no_unneeded_final_method' => true,
         'no_unused_imports' => true,
         'no_useless_else' => true,
-        'no_useless_return' => true,
         'no_whitespace_before_comma_in_array' => true,
         'no_whitespace_in_blank_line' => true,
-        'non_printable_character' => ['use_escape_sequences_in_strings' => true],
```

Comment:
👎

Reply:
Generally, I agree but it's a risky rule.

Reply:
It's risky if you're relying on non-printable characters in your code... This is meant to be sane defaults, - relying on non-printable characters doesn't really fit into sane for me. :)

Reply:
I admit I like this rule with `use_escape_sequences_in_strings` option. Will re-add.